global_translations.R

```diff
@@ -38,3 +38,34 @@ complete_trans_df <- subset(final_df, untranslated_count == 0, select = -c(fuzzy
 
 # Untranslated Packages with respective languages associated
 complete_untrans_df <- subset(final_df, translated_count == 0, select = -c(fuzzy_count))
+
+# R Language Translation Team Contact details
+library(rvest)
+library(xml2)
+library(stringr)
+
+trans_team_wbpg <- read_html("https://developer.r-project.org/TranslationTeams.html")
+
+lang <- trans_team_wbpg %>%
+  html_nodes("td:nth-child(1)") %>%
+  html_text() %>%
+  str_trim(., side = "left")
+
+lang_members <- trans_team_wbpg %>%
+  html_nodes("td+ td") %>%
+  html_text() %>%
+  str_trim(., side = "both") %>% gsub("\n", "", .)
```

Comment:
I would follow tidyverse style: one pipe call per line

https://style.tidyverse.org/pipes.html#whitespace

Reply:
and also consider using base pipe (i.e. replacing `%>%` with `|>`

Reply:
Hi Ben,
I have replaced `magrittr` piping to native piping. 
Thanks!!